install.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/sh
+
+# This install script is intended to download and install the latest available
+# release of the dep dependency manager for Golang.
+#
+# It attempts to identify the current platform and an error will be thrown if
+# the platform is not supported.
+#
+# Environment variables:
+# - INSTALL_DIRECTORY (optional): defaults to $GOPATH/bin
+# - DEP_RELEASE_TAG (optional): defaults to fetching the latest release
+# - DEP_FAKE_OS (optional): use a fake value for OS (mostly for testing)
```

Comment:
Can we use `DEP_OS`and `DEP_ARCH` instead?

Reply:
Sure, will push an update just now.